package.json

```diff
@@ -31,8 +28,9 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "eslint-plugin-sonarjs": "^0.14.0",
-    "husky": "^4.2.5",
+    "lint-staged": "^13.0.3",
```

Comment:
If we're going to add `lint-staged` to the devDependencies, should we reconsider the `lint:diff` commands across the monorepo that currently use my `config/cli/lint-diff.sh` script and maybe replace it with `lint-staged`?  I didn't look that deeply into it when I was working on my PR a while back since I was trying to avoid adding a new dependency but happy revisit since we're bringing in for this `prettier` formatting.

Reply:
Let's save it. This one is already big enough